### PR TITLE
Feature/spt 51/homepage

### PR DESCRIPTION
**Pull request by sudosf**

### CODE REVIEW

File: frontend/css/style.css [hunk 1]
```diff
@@ -1,28 +1,123 @@
...
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 4rem;
+  font-weight: 700;
+}
+
+h2 {
+
```
Comment (arinaho-muleba) [inline frontend/css/style.css]: Why are these empty?

Comment (sudosf) [reply frontend/css/style.css]: will add exact sizes later, just a template for now

**Review by arinaho-muleba — APPROVED** [review]
Happy

File: frontend/assets/images/amazing.svg
Comment (tebogo-bbd) [inline frontend/assets/images/amazing.svg:1]: why are images svgs ? png is better

File: frontend/css/style.css [hunk 2]
```diff
@@ -1,28 +1,107 @@
...
+  color: var(--text-color);
+  background-color: var(--bg-color);
+  font-family: 'Comic Neue', cursive, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 4rem;
```
Comment (tebogo-bbd) [inline frontend/css/style.css:44]: add in comment so we know what 4rem is in px

File: frontend/css/style.css
Comment (tebogo-bbd) [inline frontend/css/style.css:1]: missing h2,h3,h4,h5,h6 font sizes

**Review by tebogo-bbd — CHANGES_REQUESTED** [review]

**Review by tebogo-bbd — APPROVED** [review]

**Pull request merged**